includes/thema-taxonomy.php

```diff
@@ -118,38 +122,42 @@
  * @param String $thema_name Specific term name/slug to query
  * @param Array $thema_args Specific term query Arguments to use
  */
-function gc_get_thema_terms( $thema_name = null, $term_args = null ) {
-	$thema_taxonomy = TAX_THEMA;
-	$thema_terms    = [];
-	$thema_query    = is_array( $term_args ) ? $term_args : [
-		'taxonomy'   => $thema_taxonomy,
-		// We also want Terms with NO linked content, in this case
-		'hide_empty' => false
-	];
 
-	// Query specific term name 
-	if ( ! empty( $thema_name ) ) {
-		// If we find a Space, or an Uppercase letter, we assume `name`
-		// otherwise we use `slug`
-		$RE_disqualify_slug              = "/[\sA-Z]/";
-		$query_prop_type                 = preg_match( $RE_disqualify_slug, $thema_name ) ? 'name' : 'slug';
-		$thema_query[ $query_prop_type ] = $thema_name;
-	}
+if ( ! function_exists( 'gc_get_thema_terms' ) ) {
```

Comment:
Hiermee check je dus feitelijk andersom: als de functie al bekend is (lees: uit het gc-2020 thema) dan wordt die genegeerd. 

Echter: we willen een dergelijke check (straks) eigenlijk andersom: in het _thema_ checken of de functie al beschikbaar is (vanuit je plugin Class). Maar: dan moet die functie wel global beschikbaar zijn en ik twijfel daarover omdat hij meekomt met een `require` in een Class method...

Reply:
Ik heb in includes/thema-taxonomy.php alle onzinnige checks op 'if not function exists' weggehaald, omdat dit uiteindelijk onzin is. Die checks horen daar niet; de functies horen onderdeel te zijn van de class en de beschikbaarheid van die functies moet niet hier gecheckt worden maar in het theme.